js/views/userPageVw.js

```diff
@@ -307,6 +307,25 @@ module.exports = baseVw.extend({
           self.model.set({ownPage: self.options.ownPage});
           self.render();
           !self.currentItemHash && self.loadingDeferred.resolve();
+          
+          // Handle was requested
+          if (self.model.get('page').profile.handle) {
```

Comment:
I suppose the intent is to auto-complete the handles of nodes you've visited, vs. auto-complete based on what you typed.

If we did it in the router, it would record handles you tried to reach that didn't resolve, which you might want to try again later. Having something you know you've typed not show up in the suggestions could look like a bug. 

On the other hand, it's useful to not suggest handles if we don't think they are valid, which will also prevent typos from becoming suggestions.